tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -86,54 +92,61 @@ def test_extract_documentation_CPP(self):
-                              " @returns Exit code.\n"
-                              "          Or any other number.\n"),
-                             docstyle_CPP_doxygen.markers[0],
-                             TextRange.from_values(4, 1, 8, 4)),
-                          DocumentationComment(
-                              (" foobar\n"
-                               " @param xyz\n"),
-                              docstyle_CPP_doxygen.markers[0],
-                              TextRange.from_values(15, 1, 17, 4)),
-                          DocumentationComment(
-                              " Some alternate style of documentation\n",
-                              docstyle_CPP_doxygen.markers[4],
-                              TextRange.from_values(22, 1, 23, 1)),
-                          DocumentationComment(
-                              " ends instantly",
-                              docstyle_CPP_doxygen.markers[0],
-                              TextRange.from_values(26, 5, 26, 26)),
-                          DocumentationComment(
-                              (" Should work\n"
-                               "\n"
-                               " even without a function standing below.\n"
-                               "\n"
-                               " @param foo WHAT PARAM PLEASE!?\n"),
-                              docstyle_CPP_doxygen.markers[4],
-                              TextRange.from_values(32, 1, 37, 1))))
+        self.assertEqual(tuple(
+            extract_documentation(data, "CPP", "doxygen")),
+            (DocumentationComment(
+                ("\n"
+                 " This is the main function.\n"
+                 " @returns Exit code.\n"
+                 "          Or any other number.\n"),
+                "CPP", "doxygen",
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(4, 1, 8, 4)),
+             DocumentationComment(
+                (" foobar\n"
+                 " @param xyz\n"),
+                "CPP", "doxygen",
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(15, 1, 17, 4)),
+             DocumentationComment(
+                " Some alternate style of documentation\n",
+                "CPP", "doxygen",
+                docstyle_CPP_doxygen.markers[4],
+                TextRange.from_values(22, 1, 23, 1)),
+             DocumentationComment(
+                " ends instantly",
+                "CPP", "doxygen",
+                docstyle_CPP_doxygen.markers[0],
+                TextRange.from_values(26, 5, 26, 26)),
+             DocumentationComment(
+                (" Should work\n"
+                 "\n"
+                 " even without a function standing below.\n"
+                 "\n"
+                 " @param foo WHAT PARAM PLEASE!?\n"),
+                "CPP", "doxygen",
+                docstyle_CPP_doxygen.markers[4],
+                TextRange.from_values(32, 1, 37, 1))))
```

Comment:
would you mind move the alignment into the pep8 change? as you have a dedicated commit for that^^